lib/irb.rb

```diff
@@ -859,11 +860,12 @@ def output_value(omit = false) # :nodoc:
           end
         end
       end
+
       if multiline_p && @context.newline_before_multiline_output?
-        printf @context.return_format, "\n#{str}"
-      else
-        printf @context.return_format, str
+        str = "\n" + str
       end
+
+      Pager.page_content(format(@context.return_format, str))
```

Comment:
After quit `less`, previous output is disappeared.
```ruby
irb(main):001> 1000.times.to_a
irb(main):002> _.size
=> 1000
```
I think this disappearing feature is useful for some command's pager output but not for inspect output.

Pry uses `-R` `-X` and `-F` option

```
$ less --help
  -F  ........  --quit-if-one-screen
                  Quit if entire file fits on first screen.
  -X  ........  --no-init
                  Don't use termcap init/deinit strings.
```

Reply:
Yeah I initially thought it's good to not show the paged long output as it could push up the history a lot. But in the case of smaller terminal, that could be annoying instead.

But I'm not sure if we should use `-X` to implement the behaviour. If we were to support remote IRB, we need to centralise the output to a specific IO object, which may not be stdout.

Will push my alternative idea shortly.